src/ui/views/ViewSubscriptionNeeded.qml

```diff
@@ -170,21 +170,31 @@ VPNFlickable {
                 }
             }
 
+            VPNFooterLink {
+                id: restorePurchase
+                visible: Qt.platform.os == "ios"
+
+                // Already a subscriber?
+                labelText: qsTrId("vpn.updates.alreadyASubscriber")
+                anchors.top: grid.bottom
```

Comment:
We won't need L170 once we change the component to a `VPNLinkButton` since this is inside a `ColumnLayout{}`. Will probably need to add `Layout.alignment: Qt.AlignHCenter`.